foundations/javascript_basics/project_rock_paper_scissors.md

```diff
@@ -1,42 +1,54 @@
-
-    ~~~javascript
-    function playRound(playerSelection, computerSelection) {
-      // your code here!
-    }
+Remember to commit early and often! To refresh your memory, check the [commit messages lesson](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/commit-messages).
+
+1. Create a new Git repository for your project.
+1. Create a blank HTML document with a script tag.
+   1. We recommend writing the code in an external JavaScript file and linking it to the HTML file. You play this game entirely on the console, so don't worry about writing anything else in the HTML file.
+1. Your game plays against the computer—create a function to `return` the computer's choice:
+   1. Create a new function called `getComputerChoice`.
+   1. Make `getComputerChoice` *randomly* `return` one of the following string values: `"Rock"`, `"Paper"` or `"Scissors"`.
+   1. Remember to test what your function returns with [browser's developer tools](https://www.theodinproject.com/lessons/foundations-javascript-developer-tools) or `console.log`.
+1. Your game plays turn by turn—create a function to take the human and computer choices, then `return` the round winner:
+   1. Create a new function called `playRound`.
+   1. Define two parameters on `playRound`: `playerSelection` and `computerSelection`. Use those two parameters to take the human and computer choices as arguments.
+   1. Make `playRound` `return` a string value representing the round winner, such as: `"You Lose! Paper beats Rock"`.
+
+   ~~~javascript
+   function playRound(playerSelection, computerSelection) {
+   // your code here!
+   }
  
-    const playerSelection = "rock";
-    const computerSelection = getComputerChoice();
-    console.log(playRound(playerSelection, computerSelection));
-    ~~~
-
-6.  Write a NEW function called `game()`. Use the previous function _inside_ of this one to play a 5 round game that keeps score and reports a winner or loser at the end.
-    *   You have not officially learned how to "loop" over code to repeat function calls... if you already know about loops from somewhere else (or if you feel like doing some more learning) feel free to use them. If not, don't worry! Just call your `playRound` function 5 times in a row. Loops are covered in the next lesson.
-    *   At this point you should be using `console.log()` to display the results of each round and the winner at the end.
-    *   Use `prompt()` to get input from the user. [Read the docs here if you need to.](https://developer.mozilla.org/en-US/docs/Web/API/Window/prompt)
-    *   Feel free to re-work your previous functions if you need to. Specifically, you might want to change the return value to something more useful.
-    *   Feel free to create more "helper" functions if you think it would be useful.
+   const playerSelection = "rock";
+   const computerSelection = getComputerChoice();
+   console.log(playRound(playerSelection, computerSelection));
```

Comment:
```suggestion
   console.log(playRound(playerSelection, computerSelection));
```

the whole code block should go below last list item or place it back into it's own numbered list step to test our one round.